data/adapters/favor_adapter.py

```diff
@@ -224,7 +229,7 @@ def process_file(self):
                 hgvs = build_hgvs_from_spdi(spdi)
 
                 to_json = {
-                    '_key': id,
+                    '_key': allele_vrs_digest,
```

Comment:
Do we use spdi as _key if spdi is short enought?

Reply:
No.

Reply:
Right? Why would we?